matron/src/device/device_monitor.c

```diff
@@ -228,38 +227,35 @@ void rm_dev(struct udev_device *dev, int dev_file) {
 }
 
 void rm_dev_tty(struct udev_device *dev, const char *node) {
-#ifdef DEVICE_MONITOR_DEBUG
     fprintf(stderr, "rm_dev_tty: %s\n", node);
```

Comment:
are these prints still useful?

Reply:
it's a bit verbose, probably only useful in the short term to confirm with users that things are working. we can/should remove them later?